src/codegen/codegen_info.hpp

```diff
@@ -253,6 +253,10 @@ struct CodegenInfo {
     /// typically equal to number of primes
     int num_equations = 0;
 
+    /// True if we have to emit CVODE code
+    /// TODO: Figure out when this needs to be true
+    bool emit_cvode = false;
+
```

Comment:
Yes indeed. We couldn't figure this out with Michael from a quick look so it needs more work. Probably it can be fixed in the next PR :slightly_smiling_face: